app/default-files/default-themes/simple/partials/footer.hbs

```diff
@@ -3,48 +3,55 @@
         <div class="footer__social">
             {{#if @config.custom.socialFacebook}}
                 <a href="{{@config.custom.socialFacebook}}" aria-label="Facebook">
+                    <desc>Facebook</desc>
                     <svg>
                         <use xlink:href="{{@website.assetsUrl}}/svg/svg-map.svg#facebook"/>
                     </svg>
                 </a>
             {{/if}}
             {{#if @config.custom.socialTwitter}}
                 <a href="{{@config.custom.socialTwitter}}" aria-label="Twitter">
+                    <desc>Twitter</desc>
                     <svg>
                         <use xlink:href="{{@website.assetsUrl}}/svg/svg-map.svg#twitter"/>
                     </svg>
                 </a>
             {{/if}}
             {{#if @config.custom.socialInstagram}}
                 <a href="{{@config.custom.socialInstagram}}" aria-label="Instagram">
+                    <desc>Instagram</desc>
                     <svg>
                         <use xlink:href="{{@website.assetsUrl}}/svg/svg-map.svg#instagram"/>
                     </svg>
                 </a>
             {{/if}}
             {{#if @config.custom.socialLinkedin}}
                 <a href="{{@config.custom.socialLinkedin}}" aria-label="LinkedIn">
+                    <desc>LinkedIn</desc>
                     <svg>
                         <use xlink:href="{{@website.assetsUrl}}/svg/svg-map.svg#linkedin"/>
                     </svg>
                 </a>
             {{/if}}
             {{#if @config.custom.socialVimeo}}
                 <a href="{{@config.custom.socialVimeo}}" aria-label="Vimeo">
+                    <desc>Vimeo</desc>
                     <svg>
                         <use xlink:href="{{@website.assetsUrl}}/svg/svg-map.svg#vimeo"/>
                     </svg>
                 </a>
             {{/if}}
             {{#if @config.custom.socialPinterest}}
                 <a href="{{@config.custom.socialPinterest}}" aria-label="Pinterest">
+                    <desc>Pinterest</desc>
                     <svg>
                         <use xlink:href="{{@website.assetsUrl}}/svg/svg-map.svg#pinterest"/>
                     </svg>
                 </a>
             {{/if}}
             {{#if @config.custom.socialYoutube}}
                 <a href="{{@config.custom.socialYoutube}}" aria-label="Youtube">
+                    <desc>Youtube</desc>
```

Comment:
```suggestion
                    <desc>YouTube</desc>
```

Reply:
An <desc> element is misused, it is a description "component" of SVG file and must be used inside e.g.:
`<a href="{{@config.custom.socialPinterest}}" aria-label="Pinterest">
    <svg>
      <desc>Pinterest</desc>
      <use xlink:href="{{@website.assetsUrl}}/svg/svg-map.svg#pinterest"/>
   </svg>
</a>`

Reply:
Of course you're right. I don't know how I made that mistake. I think I wrote it in the correct way in my own theme, then tried to backport it to the default theme and just didn't notice I was one level off.